src/Parse/ParseUser.php

```diff
@@ -215,8 +215,8 @@ public static function logInWithTwitter(
         $consumer_key,
         $consumer_secret = null,
         //@codingStandardsIgnoreLine
```

Comment:
Can you remove the comment `//@codingStandardsIgnoreLine` this is preventing the linter from catching this deprecation issue.